.pylintrc

```diff
@@ -0,0 +1,53 @@
+# For documentation about this config, see
+# https://pylint.readthedocs.io/en/stable/user_guide/configuration/all-options.html (as of writing, version 2.17.4)
+
```

Comment:
Are there any changes to the default `.pylintrc` from the github-workflows repo? If not, this file can be removed and added to `.gitignore` file